tests/algorithms/exaflow/compute_average.py

```diff
@@ -0,0 +1,57 @@
+from exareme2.algorithms.exaflow.algorithm import Algorithm
+from exareme2.algorithms.exaflow.exaflow_registry import exaflow_udf
+from exareme2.algorithms.specifications import AlgorithmName
+from exareme2.worker_communication import ColumnDataFloat
+from exareme2.worker_communication import ColumnDataStr
+from exareme2.worker_communication import TabularDataResult
+
+ALGORITHM_NAME = AlgorithmName.COMPUTE_AVERAGE
```

Comment:
I checked again and we only add algorithm names there that are NOT in the test folder, so you can change this back.

Sorry 😟